package/tutorials/tutorial_peptidesim.ipynb

```diff
@@ -0,0 +1,1075 @@
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "peptide_tutorial/peptidesim_nvt-378b483c\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(ps.sims[-1].location)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### !!!change the diretory to the latest simulation folder (the output of `ps.sims[-1].location)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/gpfs/fs1/home/damirkul/ab42_fragments/tutorials/peptide_tutorial/peptidesim_nvt-378b483c\n"
+     ]
+    }
+   ],
+   "source": [
+    "cd peptide_tutorial/peptidesim_nvt-378b483c"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "total 26704\r\n",
+      "-rw-------   1 damirkul damirkul 24891900 Mar  9 13:38 #traj.trr.1#\r\n",
+      "-rw-------   1 damirkul damirkul    41939 Mar  9 15:11 index.ndx\r\n",
+      "-rw-------   1 damirkul damirkul    65868 Mar  9 15:19 #peptidesim_nvt.tpr.1#\r\n",
+      "-rw-------   1 damirkul damirkul     1937 Mar  9 15:20 bck.0.colvar\r\n",
+      "-rw-------   1 damirkul damirkul   811932 Mar  9 15:20 #traj.trr.2#\r\n",
+      "-rw-------   1 damirkul damirkul   141308 Mar  9 15:20 #peptidesim_nvt.gro.1#\r\n",
+      "-rw-------   1 damirkul damirkul     1412 Mar  9 15:20 #ener.edr.1#\r\n",
+      "-rw-------   1 damirkul damirkul    29901 Mar  9 15:20 #md.log.1#\r\n",
+      "-rw-------   1 damirkul damirkul      232 Mar  9 16:17 peptidesim_base.mdp\r\n",
+      "-rw-------   1 damirkul damirkul      370 Mar  9 16:17 peptidesim_nvt.mdp\r\n",
+      "-rw-------   1 damirkul damirkul    10990 Mar  9 16:17 mdout.mdp\r\n",
+      "-rw-------   1 damirkul damirkul    65868 Mar  9 16:17 peptidesim_nvt.tpr\r\n",
+      "-rw-------   1 damirkul damirkul     1937 Mar  9 16:17 colvar\r\n",
+      "-rw-------   1 damirkul damirkul    50640 Mar  9 16:17 state.cpt\r\n",
+      "-rw-------   1 damirkul damirkul    50640 Mar  9 16:17 state_prev.cpt\r\n",
+      "-rw-------   1 damirkul damirkul   811932 Mar  9 16:17 traj.trr\r\n",
+      "-rw-------   1 damirkul damirkul   141308 Mar  9 16:17 peptidesim_nvt.gro\r\n",
+      "-rw-------   1 damirkul damirkul     1412 Mar  9 16:17 ener.edr\r\n",
+      "-rw-------   1 damirkul damirkul    29902 Mar  9 16:17 md.log\r\n",
+      "drwx------+ 10 damirkul damirkul    16384 Mar  9 16:21 \u001b[0m\u001b[38;5;27mpeptide_tutorial\u001b[0m/\r\n",
+      "-rw-------   1 damirkul damirkul       64 Mar  9 17:09 plumed.dat\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "ls -lrt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6) Viewing a simulation with MDAnalysis and nglview"
```

Comment:
Needs another # to keep all the ordered list font size consistent. Also, please add period to the end of the sentences (many are missing)